extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/artnet-bridge.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- ArtNet Bridge -->
+    <bridge-type id="artnet-bridge">
+        <label>ArtNet Bridge</label>
+        <description>The ArtNet bridge represents a single DMX universe connected via ArtNet, only unicast</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
```

Comment:
Ok, thanks for the deep dive. So actually, this "bridge" is not a single physical device (which is the normal intention when modelling an ESH bridge), but rather already a logical aggregation of different devices on the network that are supposed to interact like if they were a single device.
So then it is up to you whether you want to change it to multiple=true or keep it as a comma-separated list (to stay with a similar approach as with the channel definitions). But you would definitely have to update the parameter description.